sdk/keyvault/azure-keyvault-secrets/tests/test_secrets_client.py

```diff
@@ -350,3 +350,28 @@ class _CustomHookPolicy(object):
     @KeyVaultClientPreparer(client_kwargs={"custom_hook_policy": _CustomHookPolicy()})
     def test_custom_hook_policy(self, client, **kwargs):
         assert isinstance(client._client._config.custom_hook_policy, SecretClientTests._CustomHookPolicy)
+
+    @ResourceGroupPreparer(random_name_enabled=True)
+    @KeyVaultPreparer()
+    @KeyVaultClientPreparer()
+    def test_continuation_token(self, client, **kwargs):
+        secret_name = "crud-secret"
```

Comment:
We may eventually share vaults between tests. Can you future-proof this a little with `create_random_name`?